print_format/request_for_quotation.jinja

```diff
@@ -8,12 +8,18 @@
     }
 </style>
 
-{% set company_address_name = frappe.db.get_value("Dynamic Link", filters={"link_doctype": "Company", "link_name": doc.company, "parenttype": "Address"}, fieldname="parent") %}
-{% set cmp_addr = frappe.get_doc("Address", company_address_name) %}
+{% set company_addresses = frappe.get_all("Dynamic Link", filters={"link_doctype": "Company", "link_name": doc.company, "parenttype": "Address"}, fields=["parent"]) %}
+{% set cmp_addr = frappe.get_doc("Address", company_addresses[0]['parent']) %}
```

Comment:
This part doesn't need to change - actually the previous approach was better and is the way that is used in the other formats as well.

Reply:
```suggestion
{% set company_address_name = frappe.db.get_value("Dynamic Link", filters={"link_doctype": "Company", "link_name": doc.company, "parenttype": "Address"}, fieldname="parent") %}
{% set cmp_addr = frappe.get_doc("Address", company_address_name) %}
```
Alright, thanks. In this suggestion is the previous approach copied.